main.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-unused-vars */
+
+// Inititalize the seconds to zero and other variables here
```

Comment:
You don't need comments that tell other people what is happening. Assume that anyone looking at your code can read the code. You only need to comment why something is happening, and only in rare cases when you can't use simple enough code to make the why clear to others.